### PR TITLE
Added alert dialog box to activate GPS

### PR DESCRIPTION
**Pull request by Subhankar29**
Fixes issue #89. "First steps for issues.

Changes: When GPS is OFF and geolocation button is pressed, Alert dialog box pops to enable GPS.

Screenshots for the change: 
screenshot_2017-03-20-19-44-55-266

### CODE REVIEW

**Comment by Abhi2424shek** [comment]
White or cream alert box would have been better, your thoughts?

**Comment by Subhankar29** [comment]
Ok. I will do it in white.

**Comment by Subhankar29** [comment]
I have updated the color to white and black. 
screenshot_2017-03-21-18-44-54-242

File: app/src/main/java/vn/mbm/phimp/me/gallery3d/media/CropImage.java [hunk 1]
```diff
@@ -288,17 +289,51 @@ public void onCreate(Bundle icicle) {
...
+                    alertDialogBuilder.setNegativeButton("Cancel",
+                            new DialogInterface.OnClickListener() {
+                                public void onClick(DialogInterface dialog, int id) {
+                                    dialog.cancel();
+                                }
+                            });
+                    AlertDialog alert = alertDialogBuilder.create();
+                    alert.show();
+
+                }
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/gallery3d/media/CropImage.java]: show a toast in the `else` block that location service already enabled.

Comment (Subhankar29) [inline app/src/main/java/vn/mbm/phimp/me/gallery3d/media/CropImage.java]: Thanks for the input. I have added it. Please have a check.

**Comment by pa1pal** [comment]
Although it has more feature and needs improvements. So send subsequent PRs for this issue. 
I am merging it now.

**Pull request merged**